cmd/scale.go

```diff
@@ -491,3 +575,20 @@ func (sc *scaleCmd) drainNodes(kubeConfig string, vmsToDelete []string) error {
 
 	return nil
 }
+
+func (sc *scaleCmd) printScaleTargetEqualsExisting(currentNodeCount int) {
+	var printNodes bool
+	trailingChar := "."
+	if sc.nodes != nil {
+		printNodes = true
+		trailingChar = ":"
+	}
+	log.Infof("Node pool %s is already at the desired count %d%s", sc.agentPoolToScale, sc.newDesiredAgentCount, trailingChar)
+	if printNodes {
+		operations.PrintNodes(sc.nodes)
+	}
+	numNodesFromK8sAPI := len(sc.nodes)
+	if currentNodeCount != numNodesFromK8sAPI {
+		sc.logger.Warnf("The Kubernetes API reports %d vms in pool %s", numNodesFromK8sAPI, sc.agentPoolToScale)
```

Comment:
same comment here, it's not very clear what this warning means without the `currentNodeCount != numNodesFromK8sAPI` code context

Reply:
also s/`vms`/`VMs`